setup.cfg

```diff
@@ -20,6 +20,10 @@ commands =
 pyinstaller40 =
 	hook-dirs = pygame.__pyinstaller:get_hook_dirs
 
+[codespell]
+ignore-words-list = ans,appen,datas,fo,nd,mechanim,paeth,testin
+skip = ./src_c/*,*.json
```

Comment:
I'm confused what this line is meant to skip. Are all files in `src_c` being skipped here?

Reply:
Since this PR is not very large, I wouldn't mind if that happened in this PR. I think https://github.com/pygame/pygame/pull/3216 fixed most of the issues anyways, right?

Reply:
I would suggest removing this `skip` line here. I have an alternate proposal and I'll mention it in my next review